docs/spec.md

```diff
@@ -170,6 +173,74 @@ Informs the client that channels are no longer available.
 }
 ```
 
+### Client Advertise
+
+- Informs the server about available client channels.
+
+#### Fields
+
+- `op`: string `"clientAdvertise"`
+- `channels`: array of:
+  - `topic`: string
```

Comment:
I think we should include a channel id, so that the full topic name doesn't have to be provided each time a message is published